Gemfile

```diff
@@ -13,7 +13,7 @@ gem "middleman-minify-html"
 gem "middleman-autoprefixer"
 
 # Minification and optimization
-gem "middleman-imageoptim"
+# gem "middleman-imageoptim" # issues with middleman v4
```

Comment:
@sisk the update to Middleman v4 removed their built in asset pipeline in favor of a "roll your own" approach